### PR TITLE
Fix/dividends style fixes

### PR DESCRIPTION
**Pull request by Grsmto**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [x] I've added this PR's link to its Asana task(s)
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

- missing legacy layout integration

### CODE REVIEW

File: packages/new-polymath-issuer/src/components/DividendList/Presenter.tsx [hunk 1]
```diff
@@ -52,12 +54,17 @@ export const DividendListPresenter = ({
           </sc.NewDividendButton>
         </Fragment>
       ) : (
-        <sc.PlaceholderButton href={newDividendUrl}>
+        <sc.PlaceholderButton
+          RouterLink={Link}
```
Comment (RafaelVidaurre) [inline packages/new-polymath-issuer/src/components/DividendList/Presenter.tsx]: `RouterLink` is going to default to the right `Link` if left out. (react-router-dom's or redux-little-router's depending on the dApp)

Comment (RafaelVidaurre) [reply packages/new-polymath-issuer/src/components/DividendList/Presenter.tsx]: (So we should be able to remove these)

File: packages/new-polymath-ui/src/components/Table/Rows/Row/index.tsx [hunk 2]
```diff
@@ -12,15 +12,15 @@ export const Row: FC<Props> = ({ row, small }) => {
     <sc.Row
       {...row.getRowProps()}
       selected={row.isSelected}
-      hasError={!row.isValid}
+      hasError={row.isValid === false}
```
Comment (RafaelVidaurre) [inline packages/new-polymath-ui/src/components/Table/Rows/Row/index.tsx]: Why was this change required?

Comment (Grsmto) [reply packages/new-polymath-ui/src/components/Table/Rows/Row/index.tsx]: Because if `isValid` is not defined it shouldn't count has being `false`. The `isValid` property can or not be defined based on what "feature" is activated in the `Table` (here it's the `useCsvParser` feature).

Comment (RafaelVidaurre) [reply packages/new-polymath-ui/src/components/Table/Rows/Row/index.tsx]: @Grsmto but `undefined` is a falsy value. Therefore you can just do `!row.isValid` which by javascript's type coercion will become a boolean

Comment (RafaelVidaurre) [reply packages/new-polymath-ui/src/components/Table/Rows/Row/index.tsx]: Oooh, wait so we have 3 states? valid, invalid and neither?
